src/main/java/teammates/ui/webapi/QueryLogsAction.java

```diff
@@ -0,0 +1,84 @@
+package teammates.ui.webapi;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.http.HttpStatus;
+
+import com.google.logging.type.LogSeverity;
+
+import teammates.common.datatransfer.QueryLogsResults;
+import teammates.common.exception.InvalidHttpParameterException;
+import teammates.common.exception.LogServiceException;
+import teammates.common.util.Const;
+import teammates.ui.output.GeneralLogsData;
+
+/**
+ * Queries the logs from Google Cloud Logging.
```

Comment:
This class needs not know where the logs come from

Reply:
Removed Google Cloud Logging